src/coreservices.cpp

```diff
@@ -594,6 +589,13 @@ void CoreServices::finalize() {
     qDebug() << t.elapsed(false).debugMillisWithUnit() << "deleting EngineMixer";
     CLEAR_AND_CHECK_DELETED(m_pEngine);
 
+    // Destroy PlayerInfo explicitly to release the track
+    // pointers of tracks that were still loaded in decks
+    // or samplers when PlayerManager was destroyed!
+    // Do this after deleting EngineMixer which makes use of
+    // PlayerInfo in EngineRecord. Fixes #12242
```

Comment:
References to closed issues should not be needed to reason about code. Comments and code should be self-sufficient, which is the case here. That suffix doesn't provide any value. If you want to keep it in the history then add in to the Git commit message.

TODO comments with references to open issues, discussions, or whatever are desirable, though. But then better use the full URL.